mungegithub/mungers/release-note-label.go

```diff
@@ -136,6 +136,9 @@ func (r *ReleaseNoteLabel) Munge(obj *github.MungeObject) {
 	}
 
 	labelToAdd := determineReleaseNoteLabel(obj)
+	if labelToAdd == "" {
```

Comment:
As for `determineReleaseNoteLabel`, if `obj.Issue.Body == nil`, should we return `releaseNoteLabelNeeded` instead of empty string?

```go
// determineReleaseNoteLabel returns the label to be added if
// correctly implemented in the PR template.  returns nil otherwise
func determineReleaseNoteLabel(obj *github.MungeObject) string {
	if obj.Issue.Body == nil {
		return ""
	}
	potentialMatch := getReleaseNote(*obj.Issue.Body)
	return chooseLabel(potentialMatch)
}
```